### DISCUSSION TITLE
Reactive value in the form leads to form submitting

### DISCUSSION THREAD

**Discussion by fellz** [vueform/vueform#291] [Questions]
Hi
I placed button in the form

```
<button @click="showQuestionsBuilder = !showQuestionsBuilder">
      Show questions
  </button>

<ButtonElement
          name="send"
          button-type="button"
          :submits="true"
          :columns="{
            default: 12,
            sm: 6
          }"
        >
          Save
        </ButtonElement>
```
and this leads to form submitting. Why so ? and what i should do to prevent that ?

**Comment by deanCompRSA** [comment]
If a button is placed inside a form and it's type is not specified, it will be treated as a type submit. Change it to:

`<button @click="showQuestionsBuilder = !showQuestionsBuilder" type="button"> Show questions</button>`
